docs/nodes/full-storage-node.mdx

```diff
@@ -88,50 +92,84 @@ A note on ports:
   after the IP address or use the  `--core.grpc.port` flag to specify another
   port if you prefer.
 
-````mdx-code-block
-<Tabs groupId="network">
```

Comment:
Why are tabs being removed from here?

Reply:
fairly certain i was just rearranging where the tabbed section starts

Reply:
yea, they're not removed they have just been moved and cover more of the page. resolving